ReactWindows/ReactNative.Shared/UIManager/UIImplementation.cs

```diff
@@ -149,6 +149,19 @@ public Task RemoveRootViewAsync(int rootViewTag)
             }
         }
 
+        /// <summary>
+        /// Refreshes RTL/LTR direction on all root views.
+        /// </summary>
+        public void UpdateLayoutDirection()
+        {
+            // RTL/LTR is implemented through Xaml FlowDirection, so we keep Yoga oblivious of this.
+            _operationsQueue.UpdateRootViewNodesDirection();
+            if (_operationsQueue.IsEmpty())
```

Comment:
What is the purpose of calling `DispatchViewUpdates` if `_operationsQueue.IsEmpty()`?

Reply:
I also followed the model from the previous method, theoretically taking into account the case this update comes outside any UIManager batch. It's just supposed to flush this change to the operation queues in that case.

---
In reply to: [223475746](https://github.com/Microsoft/react-native-windows/pull/1994#discussion_r223475746) [](ancestors = 223475746)